package com.clinica.repository;
import com.clinica.model.Prontuario;
import org.springframework.data.jpa.repository.JpaRepository;
public interface ProntuarioRepository extends JpaRepository<Prontuario, Long> {}
